src/solvers/dg_unstructured_quad/curved_map_and_geom.jl

```diff
@@ -0,0 +1,142 @@
+# transfinite mapping formula from a point (xi, eta) in reference space [-1,1]^2 to a point
+# (x,y) in physical coordinate space for a quadrilateral element with general curved sides
+#     Alg. 98 from the blue book of Kopriva
+function transfinite_quad_map(xi, eta, GammaCurves::AbstractVector{<:GammaCurve})
+
+  Xref11, Xref12 = evaluate_at(-1.0, GammaCurves[1])
+  Xref21, Xref22 = evaluate_at( 1.0, GammaCurves[1])
+  Xref31, Xref32 = evaluate_at( 1.0, GammaCurves[3])
+  Xref41, Xref42 = evaluate_at(-1.0, GammaCurves[3])
+
+  Xcomp11, Xcomp12 = evaluate_at(xi , GammaCurves[1])
+  Xcomp21, Xcomp22 = evaluate_at(eta, GammaCurves[2])
+  Xcomp31, Xcomp32 = evaluate_at(xi , GammaCurves[3])
+  Xcomp41, Xcomp42 = evaluate_at(eta, GammaCurves[4])
```

Comment:
```suggestion
function transfinite_quad_map(xi, eta, gamma_curves::AbstractVector{<:GammaCurve})

  Xref11, Xref12 = evaluate_at(-1.0, gamma_curves[1])
  Xref21, Xref22 = evaluate_at( 1.0, gamma_curves[1])
  Xref31, Xref32 = evaluate_at( 1.0, gamma_curves[3])
  Xref41, Xref42 = evaluate_at(-1.0, gamma_curves[3])

  Xcomp11, Xcomp12 = evaluate_at(xi , gamma_curves[1])
  Xcomp21, Xcomp22 = evaluate_at(eta, gamma_curves[2])
  Xcomp31, Xcomp32 = evaluate_at(xi , gamma_curves[3])
  Xcomp41, Xcomp42 = evaluate_at(eta, gamma_curves[4])
```
Typically we use snake case for variable names

Reply:
For the Xref... Xcomp... variables I assume that these are the names in the blue book?

Reply:
yeah basically. We can rename these to be more descriptive now that I am confident that they are working

Reply:
Sounds good :+1: